### ISSUE TITLE
connect process changes active chatroom

### ISSUE THREAD

**Issue by imre84** [hexchat/hexchat#2816]
Hello.

### user story:
I use hexchat, and I autoconnect to both libera.chat and IRCnet. On both servers I autojoin certain channels. What happens when I start hexchat is hexchat tries to connect to both servers right away. However libera.chat is way quicker for me to connect. At this point I forget to wait for the IRCnet to connect (10-15 more seconds) and start to type to one of the autojoin chats. However I type blindly, mostly looking at my keyboard. During typing IRCnet tends to connect as well, at which point hexchat changes the active chat, making me type into the wrong channel, in the end resulting the second half of my message to be sent to an unintended shat of the IRCnet.

### TLDR:
Having said that I suggest not to change the displayed channel during connect/autojoin process. That behaviour would make more sense, and also it would prevent mistakes like the one detailed above.

Thank you in advance.

Imre

**Comment by imre84** [comment]
seems like there's a config for it
settings menu->settings menu option->gui->channel changer/selector->the new tab gets focus: ->never
maybe you can add a "never after the first autoconnect server autojoined the autojoin channels" option to that, that would be slightly better

**Comment by imre84** [comment]
also, when I use the above setting /join wouldn't change the channel if it's newly opened, so I need to issue the /join command twice, which is undesired

**Comment by TingPing** [comment]
Duplicate of #1872

Patches welcomed for this change.
